src/XrdCms/XrdCmsManTree.cc

```diff
@@ -79,7 +79,7 @@ int XrdCmsManTree::Connect(int nID, XrdCmsNode *nP)
 {
    static CmsDiscRequest discRequest = {{0, kYR_disc, 0, 0}};
    XrdSysMutexHelper Monitor(myMutex);
-   char mybuff[8];
+   char mybuff[16];
```

Comment:
Compiler now studies string format to understand what size buffer is required (min and max bytes). According to compiler this buffer (an many other) are too small to fit potential values. Note that in most (or all) cases `snprintf` is used thus we don't have buffer overflow, but string can be truncated.